src/zap/zapimport.cpp

```diff
@@ -2165,6 +2165,11 @@ DWORD ZapIndirectHelperThunk::SaveWorker(ZapWriter * pZapWriter)
     *(WORD *)(p + 2) = 0xC000;
     p += 4;
 
+    // orr r12, r12, #1
```

Comment:
@jkotas Thank you for comment. That approach is the first appropach that I tried, but I couldn't find where the related code is (I made this patch as a workaround). Could you let me know any information related with this? I read https://github.com/dotnet/coreclr/blob/master/Documentation/botr/readytorun-overview.md but couldn't find how it is really implemented in Core CLR.